### ISSUE TITLE
[client-keystone-auth] Script for download the binary don't work

### ISSUE THREAD

**Issue by Yasin-HH** [kubernetes/cloud-provider-openstack#2728]
<!--
**Is this a BUG REPORT or FEATURE REQUEST?**:

> /kind bug

**What happened**:
I can't download the binary of client-keystone-auth with the script
https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/keystone-auth/using-keystone-webhook-authenticator-and-authorizer.md#clientkubectl-configuration

**What you expected to happen**:
Download the client-keystone-auth binary from the script

Link of the client-keystone
https://github.com/kubernetes/cloud-provider-openstack/releases/download/v1.18.0/client-keystone-auth

**How to reproduce it**:
Actual Script
repo=kubernetes/cloud-provider-openstack
version=$(curl --silent "https://api.github.com/repos/${repo}/releases/latest" | grep '"tag_name":' | awk -F '"' '{print $4}')
curl -L https://github.com/kubernetes/cloud-provider-openstack/releases//download/${version}/client-keystone-auth -o ~/client-keystone-auth
sudo chmod u+x ~/client-keystone-auth

**Anything else we need to know?**:
The actual version is 1.31 but the client version is 1.18 it's normal ?

**Environment**:
GitHub
Bash

```[tasklist]
### Tasks
```

**Comment by kundan2707** [comment]
@Yasin-HH can you add some more detail to understand issue ?

**Comment by mariusleu** [comment]
hi @kundan2707 the `client-keystone-auth` binary is not included in the releases since v1.21 (2021) hence the download method [documented here](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/keystone-auth/using-keystone-webhook-authenticator-and-authorizer.md#clientkubectl-configuration) doesn't work.

What happenede to the `client-keystone-auth` binary?

**Comment by Yasin-HH** [comment]
Yeah @mariusleu to download the client you need to find this link

https://github.com/kubernetes/cloud-provider-openstack/releases/download/v1.18.0/client-keystone-auth

**Comment by mariusleu** [comment]
@Yasin-HH I think the point here is why isn't `client-keystone-auth` included in releases after v1.21.0 ? Even your link is v.18.0 which is very old.

While the client-keystone-auth got updates: https://github.com/kubernetes/cloud-provider-openstack/commits/master/cmd/client-keystone-auth/main.go it should also be included in the release artifacts.

**Comment by bd808** [comment]
@Yasin-HH If you add a `/reopen` comment it should be possible to get this bug report open again. I don't know that doing so will suddenly summon someone interested in helping determine if there is a bug in the docs, a bug in the release process, or if the whole concept of Keystone auth is now deprecated (a different kind of documentation bug), but it feels like it is worth a shot.
